.github/workflows/checks.yaml

```diff
@@ -15,139 +19,6 @@ on:
       - checks_requested
 
 jobs:
-  pr:
```

Comment:
Perhaps we should add a direct call to [commitlint](https://github.com/conventional-changelog/commitlint) on `merge_group` events looking at all commits since `main` on the current branch. Ideally we'd find an action that supports checking both the PR desc/title and the generated commits themselves, but AFAICT there isn't one yet